chrome/app/chrome_main_delegate.cc

```diff
@@ -851,11 +855,13 @@ void ChromeMainDelegate::PreSandboxStartup() {
         command_line.GetSwitchValueASCII(switches::kLang);
 #if defined(OS_ANDROID)
 #if defined(CASTANETS)
-    ui::MaterialDesignController::Initialize();
-    const std::string loaded_locale =
-        ui::ResourceBundle::InitSharedInstanceWithLocale(
-            locale, NULL, ui::ResourceBundle::LOAD_COMMON_RESOURCES);
-    ui::ResourceBundle::GetSharedInstance().AddDataPackFromAsset("assets/resources.pak");
+    if (base::Castanets::IsEnabled()) {
+      ui::MaterialDesignController::Initialize();
+      const std::string loaded_locale =
```

Comment:
loaded_locale is being used out of this scope. (line 911)
It can make build break during Android build.

Reply:
preparing android build, will handle this